src/main/java/edu/ucla/library/bucketeer/Constants.java

```diff
@@ -87,9 +87,75 @@
     public static final String LAMBDA_MAP = "lambda-map";
 
     /**
-     * The name of the mapping of deployed verticles.
+     * The name of the mapping of deployed verticles. HERE I'm worried this is wrong, leaving this mark to remember to
```

Comment:
I don't believe I changed the constant, I think it just jumped out at me as different from everything else (dot instead of hyphen), so I left the comment to check on it, and then forgot about it? Shall I change it to a hyphen?

Reply:
actually, there's not a lot of consistency there at all, I'll just remove the remark